### PR TITLE
fix(@fremtind/jkl-toggle-switch): design adjustments on ToggleSlider

### PR DESCRIPTION
**Pull request by Mikaila94**
affects: @fremtind/jkl-toggle-switch-react, @fremtind/jkl-toggle-switch

Adds hover-state, increases font-size, refactors colors in ToggleSlider

## 📥 Proposed changes

Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue.

## ☑️ Submission checklist

-   [x] I have read the [CONTRIBUTING](https://github.com/fremtind/jokul/blob/master/CONTRIBUTING.md) doc
-   [x] `yarn build` works locally with my changes
-   [x] I have added tests that prove my fix is effective or that my feature works
-   [x] I have added necessary documentation (if appropriate)

## 💬 Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did, what alternatives you considered, etc...

### CODE REVIEW

**Comment by Mikaila94** [comment]
Designendringer på ToggeSlider as requested fra Jørgen: https://www.figma.com/file/piNkJaje7U4cEL7OlFuSCc/Hundeforsikring?node-id=3160%3A0

**Review by lfbergee — APPROVED** [review]

File: packages/toggle-switch-react/src/ToggleSlider.tsx [hunk 1]
```diff
@@ -21,8 +22,12 @@ export const ToggleSlider: FC<Props> = ({ labels, className = "", inverted, onTo
     };
 
     return (
-        <div className={`jkl-toggle-slider__wrapper ${className}`}>
-            <label htmlFor={id} className="jkl-micro jkl-component-spacing--medium-right">
+        <div
+            className={`jkl-toggle-slider__wrapper ${className}`}
+            onMouseEnter={() => setHovered(true)}
+            onMouseLeave={() => setHovered(false)}
```
[the remark this answers is not in the document]
Comment (Mikaila94) [reply packages/toggle-switch-react/src/ToggleSlider.tsx:28]: Har sett på en løsning, kan få det til med kun CSS

File: packages/toggle-switch/toggle-slider.scss [hunk 2]
```diff
@@ -123,15 +155,17 @@ $jkl-slider-focus-ring-width: rem(2px);
             &--right {
                 color: $gra-40;
             }
+        }
+        &.jkl-toggle-slider {
             &--checked {
                 & > .jkl-toggle-slider--right {
-                    color: $hvit;
+                    color: var(--jkl-slider-pill-color--active);
```
[the remark this answers is not in the document]
Comment (Mikaila94) [reply packages/toggle-switch/toggle-slider.scss:172]: fixed

File: packages/toggle-switch/toggle-slider.scss [hunk 3]
```diff
@@ -123,15 +155,17 @@ $jkl-slider-focus-ring-width: rem(2px);
...
                 }
             }
 
             &--not-checked {
                 & > .jkl-toggle-slider--left {
-                    color: $hvit;
+                    color: var(--jkl-slider-pill-color--active);
```
[the remark this answers is not in the document]
Comment (Mikaila94) [reply packages/toggle-switch/toggle-slider.scss:179]: fixed

File: packages/toggle-switch/toggle-slider.scss [hunk 4]
```diff
@@ -23,115 +24,159 @@ $jkl-slider-rounded: rem(100px);
...
-.jkl-toggle-slider {
-    @include reset-outline;
-    background-color: transparent;
-
-    &__wrapper {
-        display: flex;
-        align-items: center;
-        width: 100%;
-    }
+.jkl-toggle-slider__wrapper {
```
Comment (Mikaila94) [inline packages/toggle-switch/toggle-slider.scss:32]: Syntes det ble kronglete å jobbe med CSS når strukturen her ikke stemte overens med HTML, så derfor denne endringen.

File: packages/toggle-switch/toggle-slider.scss [hunk 5]
```diff
@@ -94,11 +100,31 @@ $jkl-slider-focus-ring-width: rem(2px);
...
+    }
+
     &--checked {
         & > .jkl-toggle-slider--right {
             color: $svart;
             color: var(--jkl-slider-pill-color--active);
         }
+
+        & > .jkl-toggle-slider--left.jkl-toggle-slider--hover {
+            color: var(--jkl-slider-hover-color);
```
[the remark this answers is not in the document]
Comment (Mikaila94) [reply packages/toggle-switch/toggle-slider.scss]: fixed

File: packages/toggle-switch-react/src/ToggleSlider.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (Mikaila94) [reply packages/toggle-switch-react/src/ToggleSlider.tsx]: fixed

File: packages/toggle-switch/toggle-slider.scss [hunk 6]
```diff
@@ -109,11 +135,17 @@ $jkl-slider-focus-ring-width: rem(2px);
             color: $svart;
             color: var(--jkl-slider-pill-color--active);
         }
+
+        & > .jkl-toggle-slider--right.jkl-toggle-slider--hover {
+            color: var(--jkl-slider-hover-color);
```
[the remark this answers is not in the document]
Comment (Mikaila94) [reply packages/toggle-switch/toggle-slider.scss]: fixed

File: packages/toggle-switch/toggle-slider.scss [hunk 7]
```diff
@@ -23,117 +24,147 @@ $jkl-slider-rounded: rem(100px);
...
-            color: $svart;
-            color: var(--jkl-slider-pill-color--active);
+
+        &--checked {
+            & > .jkl-toggle-slider--right {
+                color: $svart;
+                color: var(--jkl-slider-pill-color--active);
+            }
+
+            @media (hover: hover) {
```
Comment (Mikaila94) [inline packages/toggle-switch/toggle-slider.scss:120]: Dette for å unngå hover-effekt på mobil etter å ha klikket på et element.

Comment (lfbergee) [reply packages/toggle-switch/toggle-slider.scss:120]: :not([data-touchnavigation]) er kanskje mer robust for dette.

**Review by lfbergee — APPROVED** [review]

**Pull request merged**